library/ix-dev/community/audiobookshelf/questions.yaml

```
@@ -0,0 +1,446 @@
+groups:
+  - name: Audiobookshelf Configuration
+    description: Configure Audiobookshelf
+  - name: User and Group Configuration
+    description: Configure User and Group for Audiobookshelf
+  - name: Network Configuration
+    description: Configure Network for Audiobookshelf
+  - name: Storage Configuration
+    description: Configure Storage for Audiobookshelf
+  - name: Resources Configuration
+    description: Configure Resources for Audiobookshelf
+
+portals:
+  web_portal:
+    protocols:
+      - "$kubernetes-resource_configmap_portal_protocol"
+    host:
+      - "$kubernetes-resource_configmap_portal_host"
+    ports:
+      - "$kubernetes-resource_configmap_portal_port"
+    path: "$kubernetes-resource_configmap_portal_path"
+
+questions:
+  - variable: audiobookshelfConfig
```

```suggestion
  - variable: TZ
    group: Audiobookshelf Configuration
    label: Timezone
    schema:
      type: string
      default: Etc/UTC
      required: true
      $ref:
        - definitions/timezone

  - variable: audiobookshelfConfig
```

I've noticed that this container supports TZ, so lets add it too!